libraries/AP_HAL_ChibiOS/hwdef/MFE-PixSurveyA1/hwdef.dat

```diff
@@ -0,0 +1,21 @@
+# hw definition file for processing by chibios_hwdef.py
+# for MFE-V3, based on fmuv3
+include ../fmuv3/hwdef.dat
+
+APJ_BOARD_ID 148
```

Comment:
We can't allocate out of this range safely; please take something like 1053

Reply:
.... also, there needs to be a patch against `Tools/AP_Bootloader/board_types.txt` to claim this ID.

Reply:
> We can't allocate out of this range safely; please take something like 1053

and I want to know what that safe range is?

Reply:
As per the comments in the file, above 1000 is allocated to ArduPillot. 

Just take the next one above 1053 :-)

Reply:
> As per the comments in the file, above 1000 is allocated to ArduPillot.
> 
> Just take the next one above 1053 :-)

Thank you, I have changed, I used 1076.